### DISCUSSION TITLE
Command "make:filament-user" is not defined. Did you mean one of these?

### DISCUSSION THREAD

**Discussion by muhameti1** [filamentphp/filament#6776] [Help]
When I run "php artisan make:filament-user" it's showing me this error :Command "make:filament-user" is not defined. Did you mean one of these?
Can anyone help me with this?

**Comment by Jefry16** [comment]
i run into the same error, did you fix it?

**Reply by leandrocfe** [reply]
`php artisan about --only=Filament`
What is the output?
